 sdk/wxconfig/wx-config-win.cpp | 2 +-
 * Fixed: wx-config failed to work right when user defines wxUSE_GUI=0 and wxWidgets has USE_GUI=1 (by stahta01)

diff --git a/sdk/wxconfig/wx-config-win.cpp b/sdk/wxconfig/wx-config-win.cpp
index 8dd311d..7421995 100644
--- a/sdk/wxconfig/wx-config-win.cpp
+++ b/sdk/wxconfig/wx-config-win.cpp
@@ -686,7 +686,7 @@ public:
         /// External libs (to wxWidgets)
 
         if (cfg["USE_GUI"] == "1")
-            if (sho["wxUSE_STC"])
+            if (cfg["USE_STC"] == "1" )
                 po["__LIB_WXSCINTILLA_p"] = addLib("wxscintilla" + po["WXDEBUGFLAG"]);
 
         if (cfg["USE_GUI"] == "1")
